### DISCUSSION TITLE
PPOCRLabel auto terminates

### DISCUSSION THREAD

**Discussion by hongruic** [PaddlePaddle/PaddleOCR#10884] [Q&A]
Hi there. I'd call ```PPOCRlabel``` , it opens, I click mouse up, it then throws the below error and terminates. After importing images in a directory and clicking anywhere also similarly has the labeller terminate itself. Hoping to get help on the issue if anyone's come across it, thanks!

```
(paddle) hrcb15@hrcb15-Nitro-AN515-54:~$ PPOCRLabel
[2023/09/12 22:23:45] ppocr WARNING: When args.layout is false, args.ocr is automatically set to false
Traceback (most recent call last):
  File "/home/hrcb15/anaconda3/envs/paddle/lib/python3.11/site-packages/PPOCRLabel/PPOCRLabel.py", line 1425, in scrollRequest
    bar.setValue(bar.value() + bar.singleStep() * units)
TypeError: setValue(self, a0: int): argument 1 has unexpected type 'float'


--------------------------------------
C++ Traceback (most recent call last):
--------------------------------------
No stack trace in paddle, may be caused by external reasons.

----------------------
Error Message Summary:
----------------------
FatalError: `Process abort signal` is detected by the operating system.
  [TimeInfo: *** Aborted at 1694528633 (unix time) try "date -d @1694528633" if you are using GNU date ***]
  [SignalInfo: *** SIGABRT (@0x3e8000213c4) received by PID 136132 (TID 0x7f4e49385440) from PID 136132 ***]

Aborted (core dumped)

```

packages:

```
aiobotocore==2.5.0
aiofiles==22.1.0
aiohttp==3.8.3
aioitertools==0.7.1
aiosignal==1.2.0
aiosqlite==0.18.0
alabaster==0.7.12
anaconda-catalogs==0.2.0
anaconda-client==1.12.0
anaconda-project==0.11.1
anyio==3.5.0
appdirs==1.4.4
argon2-cffi==21.3.0
argon2-cffi-bindings==21.2.0
arrow==1.2.3
astor==0.8.1
astroid==2.14.2
astropy==5.1
asttokens==2.0.5
async-timeout==4.0.2
atomicwrites==1.4.0
attrdict==2.0.1
attrs==22.1.0
Automat==20.2.0
autopep8==1.6.0
Babel==2.12.1
backcall==0.2.0
backports.functools-lru-cache==1.6.4
backports.tempfile==1.0
backports.weakref==1.0.post1
bce-python-sdk==0.8.90
bcrypt==3.2.0
beautifulsoup4==4.12.2
binaryornot==0.4.4
black==0.0
bleach==4.1.0
blosc2==2.0.0
bokeh==3.2.1
boltons==23.0.0
botocore==1.29.76
Bottleneck==1.3.5
brotlipy==0.7.0
cachetools==5.3.1
certifi==2023.7.22
cffi==1.15.1
chardet==4.0.0
charset-normalizer==2.0.4
click==8.0.4
cloudpickle==2.2.1
clyent==1.2.2
colorama==0.4.6
colorcet==3.0.1
comm==0.1.2
conda-content-trust==0+unknown
conda-pack==0.6.0
conda-package-handling==2.2.0
conda_package_streaming==0.9.0
conda-repo-cli==1.0.41
conda-verify==3.4.2
constantly==15.1.0
contourpy==1.0.5
cookiecutter==1.7.3
cryptography==41.0.2
cssselect==1.1.0
cssutils==2.7.1
cycler==0.11.0
Cython==3.0.2
cytoolz==0.12.0
daal4py==2023.1.1
dask==2023.6.0
datasets==2.12.0
datashader==0.15.1
datashape==0.5.4
debugpy==1.6.7
decorator==5.1.1
defusedxml==0.7.1
diff-match-patch==20200713
dill==0.3.6
distributed==2023.6.0
docstring-to-markdown==0.11
docutils==0.18.1
entrypoints==0.4
et-xmlfile==1.1.0
executing==0.8.3
fastjsonschema==2.16.2
filelock==3.9.0
flake8==6.0.0
Flask==2.2.2
flask-babel==3.1.0
fonttools==4.25.0
frozenlist==1.3.3
fsspec==2023.4.0
fst-pso==1.8.1
future==0.18.3
FuzzyTM==2.0.5
gensim==4.3.0
glob2==0.7
gmpy2==2.1.2
greenlet==2.0.1
h11==0.14.0
h5py==3.7.0
HeapDict==1.0.1
holoviews==1.17.0
httpcore==0.17.3
httpx==0.24.1
huggingface-hub==0.15.1
hvplot==0.8.4
hyperlink==21.0.0
idna==3.4
imagecodecs==2021.8.26
imageio==2.31.1
imagesize==1.4.1
imbalanced-learn==0.10.1
imgaug==0.4.0
importlib-metadata==6.0.0
incremental==21.3.0
inflection==0.5.1
iniconfig==1.1.1
intake==0.6.8
intervaltree==3.1.0
ipykernel==6.19.2
ipython==8.12.0
ipython-genutils==0.2.0
ipywidgets==8.0.4
isort==5.9.3
itemadapter==0.3.0
itemloaders==1.0.4
itsdangerous==2.0.1
jaraco.classes==3.2.1
jedi==0.18.1
jeepney==0.7.1
jellyfish==0.9.0
Jinja2==3.1.2
jinja2-time==0.2.0
jmespath==0.10.0
joblib==1.2.0
json5==0.9.6
jsonpatch==1.32
jsonpointer==2.1
jsonschema==4.17.3
jupyter==1.0.0
jupyter_client==7.4.9
jupyter-console==6.6.3
jupyter_core==5.3.0
jupyter-events==0.6.3
jupyter-server==1.23.4
jupyter_server_fileid==0.9.0
jupyter_server_ydoc==0.8.0
jupyter-ydoc==0.2.4
jupyterlab==3.6.3
jupyterlab-pygments==0.1.2
jupyterlab_server==2.22.0
jupyterlab-widgets==3.0.5
keyring==23.13.1
kiwisolver==1.4.4
lazy_loader==0.2
lazy-object-proxy==1.6.0
libarchive-c==2.9
libmambapy==1.4.1
linkify-it-py==2.0.0
llvmlite==0.40.0
lmdb==1.4.1
locket==1.0.0
lxml==4.9.1
lz4==4.3.2
Markdown==3.4.1
markdown-it-py==2.2.0
MarkupSafe==2.1.1
matplotlib==3.7.1
matplotlib-inline==0.1.6
mccabe==0.7.0
mdit-py-plugins==0.3.0
mdurl==0.1.0
miniful==0.0.6
mistune==0.8.4
mkl-fft==1.3.6
mkl-random==1.2.2
mkl-service==2.4.0
more-itertools==8.12.0
mpmath==1.3.0
msgpack==1.0.3
multidict==6.0.2
multipledispatch==0.6.0
multiprocess==0.70.14
munkres==1.1.4
mypy-extensions==0.4.3
nbclassic==0.5.5
nbclient==0.5.13
nbconvert==6.5.4
nbformat==5.7.0
nest-asyncio==1.5.6
networkx==3.1
nltk==3.8.1
notebook==6.5.4
notebook_shim==0.2.2
numba==0.57.0
numexpr==2.8.4
numpy==1.23.5
numpydoc==1.5.0
opencv-contrib-python==4.8.0.76
opencv-python-headless==4.8.0.76
openpyxl==3.0.10
opt-einsum==3.3.0
packaging==23.0
paddle-bfloat==0.1.7
paddleocr==2.6.0.1
paddlepaddle==2.5.1
pandas==1.5.3
pandocfilters==1.5.0
panel==1.2.1
param==1.13.0
parsel==1.6.0
parso==0.8.3
partd==1.2.0
pathlib==1.0.1
pathspec==0.10.3
patsy==0.5.3
pep8==1.7.1
pexpect==4.8.0
pickleshare==0.7.5
Pillow==9.4.0
pip==23.2.1
pkginfo==1.9.6
platformdirs==2.5.2
plotly==5.9.0
pluggy==1.0.0
ply==3.11
pooch==1.4.0
poyo==0.5.0
PPOCRLabel==2.1.3
premailer==3.10.0
prometheus-client==0.14.1
prompt-toolkit==3.0.36
Protego==0.1.16
protobuf==4.24.2
psutil==5.9.0
ptyprocess==0.7.0
pure-eval==0.2.2
py-cpuinfo==8.0.0
pyarrow==11.0.0
pyasn1==0.4.8
pyasn1-modules==0.2.8
pyclipper==1.3.0.post4
pycodestyle==2.10.0
pycosat==0.6.4
pycparser==2.21
pycryptodome==3.18.0
pyct==0.5.0
pycurl==7.45.2
PyDispatcher==2.0.5
pydocstyle==6.3.0
pyerfa==2.0.0
pyflakes==3.0.1
pyFUME==0.2.25
Pygments==2.15.1
PyJWT==2.4.0
pylint==2.16.2
pylint-venv==2.3.0
pyls-spyder==0.4.0
PyMuPDF==1.21.1
pyodbc==4.0.34
pyOpenSSL==23.2.0
pyparsing==3.0.9
PyQt5==5.15.9
PyQt5-Qt5==5.15.2
PyQt5-sip==12.11.0
PyQtWebEngine==5.12
PyQtWebEngine-Qt5==5.15.2
pyrsistent==0.18.0
PySocks==1.7.1
pytest==7.4.0
python-dateutil==2.8.2
python-json-logger==2.0.7
python-lsp-black==1.2.1
python-lsp-jsonrpc==1.0.0
python-lsp-server==1.7.2
python-slugify==5.0.2
python-snappy==0.6.1
pytoolconfig==1.2.5
pytz==2022.7
pyviz-comms==2.3.0
PyWavelets==1.4.1
pyxdg==0.27
PyYAML==6.0
pyzmq==23.2.0
QDarkStyle==3.0.2
qstylizer==0.2.2
QtAwesome==1.2.2
qtconsole==5.4.2
QtPy==2.2.0
queuelib==1.5.0
rapidfuzz==3.2.0
rarfile==4.0
regex==2022.7.9
requests==2.31.0
requests-file==1.5.1
requests-toolbelt==1.0.0
responses==0.13.3
rfc3339-validator==0.1.4
rfc3986-validator==0.1.1
rope==1.7.0
Rtree==1.0.1
ruamel.yaml==0.17.21
ruamel-yaml-conda==0.17.21
s3fs==2023.4.0
sacremoses==0.0.43
scikit-image==0.20.0
scikit-learn==1.3.0
scikit-learn-intelex==20230426.111612
scipy==1.10.1
Scrapy==2.8.0
seaborn==0.12.2
SecretStorage==3.3.1
Send2Trash==1.8.0
service-identity==18.1.0
setuptools==68.0.0
shapely==2.0.1
simpful==2.11.0
sip==6.6.2
six==1.16.0
smart-open==5.2.1
sniffio==1.2.0
snowballstemmer==2.2.0
sortedcontainers==2.4.0
soupsieve==2.4
Sphinx==5.0.2
sphinxcontrib-applehelp==1.0.2
sphinxcontrib-devhelp==1.0.2
sphinxcontrib-htmlhelp==2.0.0
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.3
sphinxcontrib-serializinghtml==1.1.5
spyder==5.4.3
spyder-kernels==2.4.3
SQLAlchemy==1.4.39
stack-data==0.2.0
statsmodels==0.14.0
sympy==1.11.1
tables==3.8.0
tabulate==0.8.10
TBB==0.2
tblib==1.7.0
tenacity==8.2.2
terminado==0.17.1
text-unidecode==1.3
textdistance==4.2.1
threadpoolctl==2.2.0
three-merge==0.1.1
tifffile==2021.7.2
tinycss2==1.2.1
tldextract==3.2.0
tokenizers==0.13.2
toml==0.10.2
tomlkit==0.11.1
toolz==0.12.0
torch==2.0.1
torchaudio==2.0.2
torchvision==0.15.2
tornado==6.3.2
tqdm==4.65.0
traitlets==5.7.1
transformers==4.29.2
trash-cli==0.23.2.13.2
Twisted==22.10.0
typing==3.7.4.3
typing_extensions==4.7.1
uc-micro-py==1.0.1
ujson==5.4.0
Unidecode==1.2.0
urllib3==1.26.16
visualdl==2.5.3
w3lib==1.21.0
watchdog==2.1.6
wcwidth==0.2.5
webencodings==0.5.1
websocket-client==0.58.0
Werkzeug==2.2.3
whatthepatch==1.0.2
wheel==0.38.4
widgetsnbextension==4.0.5
wrapt==1.14.1
wurlitzer==3.0.2
xarray==2023.6.0
xlrd==1.2.0
xxhash==2.0.2
xyzservices==2022.9.0
y-py==0.5.9
yapf==0.31.0
yarl==1.8.1
ypy-websocket==0.8.2
zict==2.2.0
zipp==3.11.0
zope.interface==5.4.0
zstandard==0.19.0
```

**Comment by kishore-work-hard** [comment]
> Hi there. I'd call `PPOCRlabel` , it opens, I click mouse up, it then throws the below error and terminates. After importing images in a directory and clicking anywhere also similarly has the labeller terminate itself. Hoping to get help on the issue if anyone's come across it, thanks!
> 
> ```
> (paddle) hrcb15@hrcb15-Nitro-AN515-54:~$ PPOCRLabel
> [2023/09/12 22:23:45] ppocr WARNING: When args.layout is false, args.ocr is automatically set to false
> Traceback (most recent call last):
>   File "/home/hrcb15/anaconda3/envs/paddle/lib/python3.11/site-packages/PPOCRLabel/PPOCRLabel.py", line 1425, in scrollRequest
>     bar.setValue(bar.value() + bar.singleStep() * units)
> TypeError: setValue(self, a0: int): argument 1 has unexpected type 'float'
> 
> 
> --------------------------------------
> C++ Traceback (most recent call last):
> --------------------------------------
> No stack trace in paddle, may be caused by external reasons.
> 
> ----------------------
> Error Message Summary:
> ----------------------
> FatalError: `Process abort signal` is detected by the operating system.
>   [TimeInfo: *** Aborted at 1694528633 (unix time) try "date -d @1694528633" if you are using GNU date ***]
>   [SignalInfo: *** SIGABRT (@0x3e8000213c4) received by PID 136132 (TID 0x7f4e49385440) from PID 136132 ***]
> 
> Aborted (core dumped)
> ```
> 
> packages:
> 
> ```
> aiobotocore==2.5.0
> aiofiles==22.1.0
> aiohttp==3.8.3
> aioitertools==0.7.1
> aiosignal==1.2.0
> aiosqlite==0.18.0
> alabaster==0.7.12
> anaconda-catalogs==0.2.0
> anaconda-client==1.12.0
> anaconda-project==0.11.1
> anyio==3.5.0
> appdirs==1.4.4
> argon2-cffi==21.3.0
> argon2-cffi-bindings==21.2.0
> arrow==1.2.3
> astor==0.8.1
> astroid==2.14.2
> astropy==5.1
> asttokens==2.0.5
> async-timeout==4.0.2
> atomicwrites==1.4.0
> attrdict==2.0.1
> attrs==22.1.0
> Automat==20.2.0
> autopep8==1.6.0
> Babel==2.12.1
> backcall==0.2.0
> backports.functools-lru-cache==1.6.4
> backports.tempfile==1.0
> backports.weakref==1.0.post1
> bce-python-sdk==0.8.90
> bcrypt==3.2.0
> beautifulsoup4==4.12.2
> binaryornot==0.4.4
> black==0.0
> bleach==4.1.0
> blosc2==2.0.0
> bokeh==3.2.1
> boltons==23.0.0
> botocore==1.29.76
> Bottleneck==1.3.5
> brotlipy==0.7.0
> cachetools==5.3.1
> certifi==2023.7.22
> cffi==1.15.1
> chardet==4.0.0
> charset-normalizer==2.0.4
> click==8.0.4
> cloudpickle==2.2.1
> clyent==1.2.2
> colorama==0.4.6
> colorcet==3.0.1
> comm==0.1.2
> conda-content-trust==0+unknown
> conda-pack==0.6.0
> conda-package-handling==2.2.0
> conda_package_streaming==0.9.0
> conda-repo-cli==1.0.41
> conda-verify==3.4.2
> constantly==15.1.0
> contourpy==1.0.5
> cookiecutter==1.7.3
> cryptography==41.0.2
> cssselect==1.1.0
> cssutils==2.7.1
> cycler==0.11.0
> Cython==3.0.2
> cytoolz==0.12.0
> daal4py==2023.1.1
> dask==2023.6.0
> datasets==2.12.0
> datashader==0.15.1
> datashape==0.5.4
> debugpy==1.6.7
> decorator==5.1.1
> defusedxml==0.7.1
> diff-match-patch==20200713
> dill==0.3.6
> distributed==2023.6.0
> docstring-to-markdown==0.11
> docutils==0.18.1
> entrypoints==0.4
> et-xmlfile==1.1.0
> executing==0.8.3
> fastjsonschema==2.16.2
> filelock==3.9.0
> flake8==6.0.0
> Flask==2.2.2
> flask-babel==3.1.0
> fonttools==4.25.0
> frozenlist==1.3.3
> fsspec==2023.4.0
> fst-pso==1.8.1
> future==0.18.3
> FuzzyTM==2.0.5
> gensim==4.3.0
> glob2==0.7
> gmpy2==2.1.2
> greenlet==2.0.1
> h11==0.14.0
> h5py==3.7.0
> HeapDict==1.0.1
> holoviews==1.17.0
> httpcore==0.17.3
> httpx==0.24.1
> huggingface-hub==0.15.1
> hvplot==0.8.4
> hyperlink==21.0.0
> idna==3.4
> imagecodecs==2021.8.26
> imageio==2.31.1
> imagesize==1.4.1
> imbalanced-learn==0.10.1
> imgaug==0.4.0
> importlib-metadata==6.0.0
> incremental==21.3.0
> inflection==0.5.1
> iniconfig==1.1.1
> intake==0.6.8
> intervaltree==3.1.0
> ipykernel==6.19.2
> ipython==8.12.0
> ipython-genutils==0.2.0
> ipywidgets==8.0.4
> isort==5.9.3
> itemadapter==0.3.0
> itemloaders==1.0.4
> itsdangerous==2.0.1
> jaraco.classes==3.2.1
> jedi==0.18.1
> jeepney==0.7.1
> jellyfish==0.9.0
> Jinja2==3.1.2
> jinja2-time==0.2.0
> jmespath==0.10.0
> joblib==1.2.0
> json5==0.9.6
> jsonpatch==1.32
> jsonpointer==2.1
> jsonschema==4.17.3
> jupyter==1.0.0
> jupyter_client==7.4.9
> jupyter-console==6.6.3
> jupyter_core==5.3.0
> jupyter-events==0.6.3
> jupyter-server==1.23.4
> jupyter_server_fileid==0.9.0
> jupyter_server_ydoc==0.8.0
> jupyter-ydoc==0.2.4
> jupyterlab==3.6.3
> jupyterlab-pygments==0.1.2
> jupyterlab_server==2.22.0
> jupyterlab-widgets==3.0.5
> keyring==23.13.1
> kiwisolver==1.4.4
> lazy_loader==0.2
> lazy-object-proxy==1.6.0
> libarchive-c==2.9
> libmambapy==1.4.1
> linkify-it-py==2.0.0
> llvmlite==0.40.0
> lmdb==1.4.1
> locket==1.0.0
> lxml==4.9.1
> lz4==4.3.2
> Markdown==3.4.1
> markdown-it-py==2.2.0
> MarkupSafe==2.1.1
> matplotlib==3.7.1
> matplotlib-inline==0.1.6
> mccabe==0.7.0
> mdit-py-plugins==0.3.0
> mdurl==0.1.0
> miniful==0.0.6
> mistune==0.8.4
> mkl-fft==1.3.6
> mkl-random==1.2.2
> mkl-service==2.4.0
> more-itertools==8.12.0
> mpmath==1.3.0
> msgpack==1.0.3
> multidict==6.0.2
> multipledispatch==0.6.0
> multiprocess==0.70.14
> munkres==1.1.4
> mypy-extensions==0.4.3
> nbclassic==0.5.5
> nbclient==0.5.13
> nbconvert==6.5.4
> nbformat==5.7.0
> nest-asyncio==1.5.6
> networkx==3.1
> nltk==3.8.1
> notebook==6.5.4
> notebook_shim==0.2.2
> numba==0.57.0
> numexpr==2.8.4
> numpy==1.23.5
> numpydoc==1.5.0
> opencv-contrib-python==4.8.0.76
> opencv-python-headless==4.8.0.76
> openpyxl==3.0.10
> opt-einsum==3.3.0
> packaging==23.0
> paddle-bfloat==0.1.7
> paddleocr==2.6.0.1
> paddlepaddle==2.5.1
> pandas==1.5.3
> pandocfilters==1.5.0
> panel==1.2.1
> param==1.13.0
> parsel==1.6.0
> parso==0.8.3
> partd==1.2.0
> pathlib==1.0.1
> pathspec==0.10.3
> patsy==0.5.3
> pep8==1.7.1
> pexpect==4.8.0
> pickleshare==0.7.5
> Pillow==9.4.0
> pip==23.2.1
> pkginfo==1.9.6
> platformdirs==2.5.2
> plotly==5.9.0
> pluggy==1.0.0
> ply==3.11
> pooch==1.4.0
> poyo==0.5.0
> PPOCRLabel==2.1.3
> premailer==3.10.0
> prometheus-client==0.14.1
> prompt-toolkit==3.0.36
> Protego==0.1.16
> protobuf==4.24.2
> psutil==5.9.0
> ptyprocess==0.7.0
> pure-eval==0.2.2
> py-cpuinfo==8.0.0
> pyarrow==11.0.0
> pyasn1==0.4.8
> pyasn1-modules==0.2.8
> pyclipper==1.3.0.post4
> pycodestyle==2.10.0
> pycosat==0.6.4
> pycparser==2.21
> pycryptodome==3.18.0
> pyct==0.5.0
> pycurl==7.45.2
> PyDispatcher==2.0.5
> pydocstyle==6.3.0
> pyerfa==2.0.0
> pyflakes==3.0.1
> pyFUME==0.2.25
> Pygments==2.15.1
> PyJWT==2.4.0
> pylint==2.16.2
> pylint-venv==2.3.0
> pyls-spyder==0.4.0
> PyMuPDF==1.21.1
> pyodbc==4.0.34
> pyOpenSSL==23.2.0
> pyparsing==3.0.9
> PyQt5==5.15.9
> PyQt5-Qt5==5.15.2
> PyQt5-sip==12.11.0
> PyQtWebEngine==5.12
> PyQtWebEngine-Qt5==5.15.2
> pyrsistent==0.18.0
> PySocks==1.7.1
> pytest==7.4.0
> python-dateutil==2.8.2
> python-json-logger==2.0.7
> python-lsp-black==1.2.1
> python-lsp-jsonrpc==1.0.0
> python-lsp-server==1.7.2
> python-slugify==5.0.2
> python-snappy==0.6.1
> pytoolconfig==1.2.5
> pytz==2022.7
> pyviz-comms==2.3.0
> PyWavelets==1.4.1
> pyxdg==0.27
> PyYAML==6.0
> pyzmq==23.2.0
> QDarkStyle==3.0.2
> qstylizer==0.2.2
> QtAwesome==1.2.2
> qtconsole==5.4.2
> QtPy==2.2.0
> queuelib==1.5.0
> rapidfuzz==3.2.0
> rarfile==4.0
> regex==2022.7.9
> requests==2.31.0
> requests-file==1.5.1
> requests-toolbelt==1.0.0
> responses==0.13.3
> rfc3339-validator==0.1.4
> rfc3986-validator==0.1.1
> rope==1.7.0
> Rtree==1.0.1
> ruamel.yaml==0.17.21
> ruamel-yaml-conda==0.17.21
> s3fs==2023.4.0
> sacremoses==0.0.43
> scikit-image==0.20.0
> scikit-learn==1.3.0
> scikit-learn-intelex==20230426.111612
> scipy==1.10.1
> Scrapy==2.8.0
> seaborn==0.12.2
> SecretStorage==3.3.1
> Send2Trash==1.8.0
> service-identity==18.1.0
> setuptools==68.0.0
> shapely==2.0.1
> simpful==2.11.0
> sip==6.6.2
> six==1.16.0
> smart-open==5.2.1
> sniffio==1.2.0
> snowballstemmer==2.2.0
> sortedcontainers==2.4.0
> soupsieve==2.4
> Sphinx==5.0.2
> sphinxcontrib-applehelp==1.0.2
> sphinxcontrib-devhelp==1.0.2
> sphinxcontrib-htmlhelp==2.0.0
> sphinxcontrib-jsmath==1.0.1
> sphinxcontrib-qthelp==1.0.3
> sphinxcontrib-serializinghtml==1.1.5
> spyder==5.4.3
> spyder-kernels==2.4.3
> SQLAlchemy==1.4.39
> stack-data==0.2.0
> statsmodels==0.14.0
> sympy==1.11.1
> tables==3.8.0
> tabulate==0.8.10
> TBB==0.2
> tblib==1.7.0
> tenacity==8.2.2
> terminado==0.17.1
> text-unidecode==1.3
> textdistance==4.2.1
> threadpoolctl==2.2.0
> three-merge==0.1.1
> tifffile==2021.7.2
> tinycss2==1.2.1
> tldextract==3.2.0
> tokenizers==0.13.2
> toml==0.10.2
> tomlkit==0.11.1
> toolz==0.12.0
> torch==2.0.1
> torchaudio==2.0.2
> torchvision==0.15.2
> tornado==6.3.2
> tqdm==4.65.0
> traitlets==5.7.1
> transformers==4.29.2
> trash-cli==0.23.2.13.2
> Twisted==22.10.0
> typing==3.7.4.3
> typing_extensions==4.7.1
> uc-micro-py==1.0.1
> ujson==5.4.0
> Unidecode==1.2.0
> urllib3==1.26.16
> visualdl==2.5.3
> w3lib==1.21.0
> watchdog==2.1.6
> wcwidth==0.2.5
> webencodings==0.5.1
> websocket-client==0.58.0
> Werkzeug==2.2.3
> whatthepatch==1.0.2
> wheel==0.38.4
> widgetsnbextension==4.0.5
> wrapt==1.14.1
> wurlitzer==3.0.2
> xarray==2023.6.0
> xlrd==1.2.0
> xxhash==2.0.2
> xyzservices==2022.9.0
> y-py==0.5.9
> yapf==0.31.0
> yarl==1.8.1
> ypy-websocket==0.8.2
> zict==2.2.0
> zipp==3.11.0
> zope.interface==5.4.0
> zstandard==0.19.0
> ```

did you fix this ? i also have same error

**Reply by hongruic** [reply]
@kishore-work-hard no solution yet. have been stuck on this for some time

**Reply by kishore-work-hard** [reply]
its actually python version issue. u need to use older python. 3.9.19 ... i use this. works for me

**Reply by GreatV** [reply]
maybe you can try `bar.setValue(int(bar.value() + bar.singleStep() * units))`

**Comment by kishore-work-hard** [comment]
i fixed it by changing my python version. works now
